src/api/dataSet.js

```diff
@@ -57,6 +64,14 @@ export async function updateDataSet(dataSetId, dataSet, details = false) {
     : result;
 }
 
+export async function updateDataSetOptions(dataset, token = false) {
```

Comment:
The one above it wasn't taking headers and had a weird return clause. I think it's better to leave this alone for now. This will be removed anyway when we move away from redux.